Sprint-2/interpret/invert.js

```diff
@@ -10,20 +10,29 @@ function invert(obj) {
   const invertedObj = {};
 
   for (const [key, value] of Object.entries(obj)) {
-    invertedObj.key = value;
+    //console.log(key, value);
+    invertedObj[value] = key;
   }
 
   return invertedObj;
 }
+console.log(invert({a:1}));
 
 // a) What is the current return value when invert is called with { a : 1 }
+// it shold return {key:1}, as "key" is a variable here
 
 // b) What is the current return value when invert is called with { a: 1, b: 2 }
-
+// it will return:{key:2},
 // c) What is the target return value when invert is called with {a : 1, b: 2}
+//{1:"a" , 2:"b"}
 
 // c) What does Object.entries return? Why is it needed in this program?
+// it converts each object's properties to indivisual arrays
```

Comment:
What items can be found in each of these "individual arrays"?

Reply:
it is better to say that it provides a convenient way to access both keys and values of an object in a single operation, making it essential for efficiently inverting the object in this program.